integrationtests/controller/bundle/status_test.go

```diff
@@ -70,25 +70,42 @@ var _ = Describe("Bundle Status Fields", func() {
 				if err != nil {
 					return err
 				}
+				resources := []v1alpha1.BundleDeploymentResource{}
+				resource := v1alpha1.BundleDeploymentResource{
+					Kind:       "ConfigMap",
+					APIVersion: "v1",
+					Namespace:  namespace,
+					Name:       "app-config",
+				}
+				resources = append(resources, resource)
 				bd.Status.Display.State = "Ready"
 				bd.Status.AppliedDeploymentID = bd.Spec.DeploymentID
 				bd.Status.Ready = true
 				bd.Status.NonModified = true
+				bd.Status.Resources = resources
 				return k8sClient.Status().Update(ctx, bd)
 			}).ShouldNot(HaveOccurred())
 
 			err = k8sClient.Get(ctx, types.NamespacedName{Namespace: namespace, Name: "name"}, bd)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(bd.Status.Display.State).To(Equal("Ready"))
 			Expect(bd.Status.AppliedDeploymentID).To(Equal(bd.Spec.DeploymentID))
+			Expect(bd.Status.Resources).ToNot(BeNil())
```

Comment:
This looks safer to me:
```suggestion
			Expect(len(bd.Status.Resources)).To(Equal(1))
```